jax/_src/api.py

```diff
@@ -1316,7 +1316,11 @@ def _split(x, indices, axis):
     return x.split(indices, axis)
 
 
-def vmap(fun: F, in_axes=0, out_axes=0, axis_name=None, axis_size=None) -> F:
+def vmap(fun: F,
+         in_axes: Union[None, int, Sequence[Any]] = 0,
```

Comment:
I don't think `None` is a viable option here. You can have `None` within a sequence of `in_axes`, but not by itself.